src/utilities/buildClientSchema.js

```diff
@@ -238,7 +240,8 @@ export function buildClientSchema(
     if (!objectIntrospection.interfaces) {
       throw new Error(
         'Introspection result missing interfaces: ' +
-          inspect(objectIntrospection),
+          inspect(objectIntrospection) +
+          '.',
```

Comment:
It's better to have it like `Introspection result missing interfaces: ${inspect(objectIntrospection)}.` but only if it will fit in 80 chars limit and prettier will keep it as a single line. 
If not you can do:
```
const objectInstrospectionStr = inspect(objectIntrospection);
throw new Error(`Introspection result missing interfaces: ${objectInstrospectionStr}.`);
```
Please do it for all other error messages you changed except for `devCheck` since long term plan is to remove it in production build so it should be self-contained.